pad_crm/static/src/css/pad_crm.css

```diff
@@ -0,0 +1,4 @@
+.oe_pad_readonly {
```

Comment:
While it maybe doesn't affect project_pad, it might affect other modules that use ``widget="pad"``, because it's built-in class and not something you added manually. So, you need make a note about it

Reply:
added text to README